### DISCUSSION TITLE
Can not display project blocks

### DISCUSSION THREAD

**Discussion by kaizisntme** [JanDeDobbeleer/oh-my-posh#2903] [Troubleshoot]
### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

I create my own theme, and now I want to make project block. When I go to nodejs project folder, that block not display. IDK why, it's because my version not support it or something else? <br>
code:
```json
		{
			"type": "project",
			"style": "plain",
			"foreground": "",
			"properties": {
			  "display_version": true,
			  "prefix": "<#fff>on</> 📦"
			}
		}
```

### Theme

custom theme 

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
 Admin@KaizOffical   ~  master  80  ﲵ pwsh  NO CONFIG   
```

**Comment by JanDeDobbeleer** [comment]
@KaizOffical because it doesn't load the configuration. Somehow you have the wrong path added to the `--config` flag. As stated, `oh-my-posh debug` will also show what it tries to load and why that doesn't work.

**Reply by kaizisntme** [reply]
> @KaizOffical because it doesn't load the configuration. Somehow you have the wrong path added to the `--config` flag. As stated, `oh-my-posh debug` will also show what it tries to load and why that doesn't work.

so how to fix the config path?

**Reply by JanDeDobbeleer** [reply]
> As stated, `oh-my-posh debug` will also show what it tries to load and why that doesn't work
